src/Simulation/Simulators.Tests/ResourcesEstimatorTests.cs

```diff
@@ -166,14 +182,14 @@ public void VerifyTracingMultipleOperationsTest()
             Assert.Equal(1.0 + 0.0, data2.Rows.Find("T")["Sum"]);
             Assert.Equal(0.0 + 1.0, data2.Rows.Find("R")["Sum"]);
             Assert.Equal(0.0 + 1.0, data2.Rows.Find("Measure")["Sum"]);
-            Assert.Equal(2.0 + 3.0, data2.Rows.Find("Width")["Sum"]);
-            Assert.Equal(System.Math.Max(2.0, 3.0), data2.Rows.Find("Width")["Max"]);
+            Assert.Equal(2.0 + 3.0, data2.Rows.Find("ExtraWidth")["Sum"]);
```

Comment:
If Width is the metric that is compatible with Depth then it would make more sense to me to run the tracer once with depth optimization and once with width optimization and check the Depth and Width metrics for both cases instead of keeping the old numbers.

Reply:
There's a DepthVersusWidthTest for exactly that purpose. This test is for tracking width in subroutines.